src/poetry/core/masonry/utils/module.py

```diff
@@ -72,11 +72,15 @@ def __init__(
             packages = [default_package]
 
         for package in packages:
+            formats = package.get("format", ["sdist", "wheel"])
```

Comment:
wasn't the point of #773 to default this only to "sdist"?

Reply:
In my understanding this was about the behavior of format for the include keyword and not for packages. 🤷I guess @radoering can say more about it.

Reply:
can confirm that I find this confusing

so I see, #773 patched up the package and include objects to add default values if not present - but there is nothing corresponding to this in poetry's `RunCommand._module()`.

Perhaps a better way would be to kill both birds with one stone and do this defaulting at the time the package definition is read, I think in poetry-core's `Factory._configure_package_poetry_specifics`.  Then it is always available for everyone and no further special casing is needed.

Reply:
Ah, I guess you found the places I was looking for initially. I'm on mobile so I have to take a look at latet/tomorrow.